### PR TITLE
Update sharepoint-site-usage.md

### PR DESCRIPTION
**Pull request by JeremyEsposito**
Microsoft 365 Reports in the admin center is not supported for GCC High and DoD tenants. Confirmed by SPO PG and PM's.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/activity-reports/sharepoint-site-usage.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ description: "Get the SharePoint site usage report to know how many files users
 As an Microsoft 365 admin, the **Reports** dashboard shows you the activity overview across various products in your organization. It enables you to drill in to get more granular insight about the activities specific to each product. For example, you can get a high level view of the value you are getting from SharePoint in terms of the total number of files that users store in SharePoint sites, how many files are actively being used, and the storage consumed across all these sites. Then, you can drill into the SharePoint site usage report to understand the trends and per site level details for all sites. 
   
 > [!NOTE]
-> You must be a global administrator, global reader or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports. 
+> You must be a global administrator, global reader or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/sharepoint-site-usage.md:30]:
```suggestion
> You must be a global administrator, global reader, or reports reader in Microsoft 365 or an Exchange, SharePoint, Teams Service, Teams Communications, or Skype for Business administrator to see reports.
```

File: microsoft-365/admin/activity-reports/sharepoint-site-usage.md [hunk 2]
```diff
@@ -27,7 +27,8 @@ description: "Get the SharePoint site usage report to know how many files users
 As an Microsoft 365 admin, the **Reports** dashboard shows you the activity overview across various products in your organization. It enables you to drill in to get more granular insight about the activities specific to each product. For example, you can get a high level view of the value you are getting from SharePoint in terms of the total number of files that users store in SharePoint sites, how many files are actively being used, and the storage consumed across all these sites. Then, you can drill into the SharePoint site usage report to understand the trends and per site level details for all sites.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/sharepoint-site-usage.md:27]:
```suggestion
As a Microsoft 365 admin, the **Reports** dashboard shows you the activity overview across various products in your organization. It enables you to drill in to get more granular insight about the activities specific to each product. For example, you can get a high-level view of the value you are getting from SharePoint in terms of the total number of files that users store in SharePoint sites, how many files are actively being used, and the storage consumed across all these sites. Then, you can drill into the SharePoint site usage report to understand the trends and per site level details for all sites. 
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/sharepoint-site-usage.md:27]: Please also change "As an Microsoft..." to "As a Microsoft..." and "high level" to "high-level".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@JeremyEsposito - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**